src/components/BeerDetail.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+
+
+function BeerDetail(props) {
+
+    const { beerId } = useParams();
+    const [beersArr, setBeersArr] = useState([]);
+
+    useEffect(() => {
+        axios.get(`process.env.REACT_APP_API_URL/${beerId}`)
+            .then(response => {
+                props.callbackToUpdateList()
+                setBeersArr(response.data);
+            })
+            .catch(e => console.log("Error getting beers from API", e))
+    }, []);
+
+    if (!beersArr) {
+        return <div>Loading...</div>;
+    }
+
+    return (
+        <div>
+            <h1>One single beer</h1>
+            <img src={props.image_url} alt={props.name} />
```

Comment:
The same as before, but in that case we just don't get the info. You need to call the `beersArr.` for those